### PR TITLE
create rule 017

### PR DESCRIPTION
**Pull request by uche-lextego**

### CODE REVIEW

**Review by vorsterk — APPROVED** [review]
Looks good to me

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: "2"
```
Comment (johanfol) [inline docker-compose.yaml:1]: This file is unnecessary

File: global.d.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Promise method
```
Comment (johanfol) [inline global.d.ts:1]: What's this file for?

File: src/controllers/execute.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    ctx.status = 500;
+    return ctx;
+  }
+
+  try {
+    const span = apm.startSpan('handleTransaction');
+    const ruleResult = await handleTransaction(request);
+    await sendRuleResult(ruleResult, request);
+    span?.end();
+    const resultMessage = `Result for Rule 017@1.0.0 is ${ruleResult.result}`;
```
Comment (johanfol) [inline src/controllers/execute.ts:29]: this should be 017@${config.ruleVersion}

**Review by johanfol — APPROVED** [review]

File: src/controllers/execute.ts (hunk 3 above)
Comment (uche-lextego) [reply src/controllers/execute.ts]: Thanks, Foley

File: global.d.ts (hunk 2 above)
Comment (uche-lextego) [reply global.d.ts]: Removed

**Pull request merged**